### DISCUSSION TITLE
Could someone tell me how to get 7 days EPG please ?

### DISCUSSION THREAD

**Discussion by ian10951** [iptv-org/community#120] [Q&A] [answered]
Currently I'm using

wget -w 2 -qO /var/www/html/tvtv.xml 'https://iptv-org.github.io/epg/guides/us/tvtv.us.epg.xml'

ran from a crontab daily to get USA EPG, however it only ever seems to give 1 day of data which often runs out at midnight and as the cron is set to run at 6am everyday the EPG remains blank from midnight till 6am. This is a particular pain because the USA is 5 hours behind so their primetime viewing is at 2am here, meaning without any EPG I cannot set to record primetime events.

Thanks.

**Comment by BellezaEmporium** [comment, marked as answer]
This is the wrong place to post this kind of aspects. You are currently on the iptv repo, however this I believe should belong to the epg repo.

If I'm wrong, @freearhey should tell you what's going on with the EPG.
